source/password.html

```diff
@@ -1,13 +1,66 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <title>Password</title>
+
+    <link rel="stylesheet" type="text/css" href="styles/passwordPage.css">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <link rel="icon" type="image/png" href="images/infiniteIcon.png">
+    <title>Password Page</title>
 </head>
 <body>
-    <h1>Password</h1>
-    
-    <!-- Your content goes here -->
-    <a href="./index.html">Back</a>
-    <script src="scripts/password.js"></script>
+    <div class="background-layer1">
+        <div class="background-layer2">
+            <div class="background-layer3">
+                <div class="logoBox">
+                    <img src="images/infiniteLogo.jpeg" alt="team logo">
+                    <p>Infinite Loops</p>
+                </div>
+                <div class="helpBox">
+                    <img src="images/questionMark.png" alt="helpBox" class="clickable-image" id="triggerPopup">
+                    <div id="myModal" class="modal">
```

Comment:
Thanks for adding a help popup! Just make sure that the background is opaque and the text is readable on different screen sizes (at least different desktop sizes, don't worry about readability on mobile and tablet for now)